src/index.njk

```diff
@@ -90,60 +44,27 @@ layout: homepageLayout.njk
 
       <div class="govuk-grid-row">
         <div class="govuk-grid-column-full">
-          <h2 class="govuk-heading-m">Accessibility and assisted digital</h2>
+          <h2 class="govuk-heading-m">Principles we follow</h2>
 
-          <div class="govuk-grid-row">
-            <div class="govuk-grid-column-one-third">
-              <a href="https://www.gov.uk/service-manual/helping-people-to-use-your-service" class="app-card">
-                <h3 class="govuk-heading-s app-card__heading">Accessibility and assisted digital</h3>
-                <p class="govuk-body">Help and encourage all users to use HMRC services.</p>
-              </a>
-            </div>
-          </div>
-        </div>
-      </div>
+           <p class="govuk-body">
```

Comment:
Looks like you've got an opening paragraph element where there shouldn't be one. Should be remove
```suggestion
```